packages/muelu/test/unit_tests/Hierarchy.cpp

```diff
@@ -1592,11 +1592,13 @@ namespace MueLuTests {
     // Write matrices out, read fine A back in, and check that the read was ok
     // by using a matvec with a random vector.
     char t[] = "XXXXXX";
+    int fd;
```

Comment:
You could rename to `filedescriptor`.... a bit more descriptive

Reply:
done!